### PR TITLE
i18n(fr): Add `guides/backend/sentry.mdx` :tada:

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/backend/sentry.mdx` :tada:

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/backend/sentry.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    ```
+  </Fragment>
+  <Fragment slot="yarn">
+    ```shell
+    yarn astro add @sentry/astro
+    ```
+  </Fragment>
+</PackageManagerTabs>
+
+L'astro CLI installe le paquet SDK et ajoute l'intégration Sentry à votre fichier `astro.config.mjs`.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/backend/sentry.mdx:47]: I think "astro CLI" should be replaced with "Le CLI d'Astro" to be consistent with the [following page title](https://docs.astro.build/fr/reference/cli-reference/) in the nav.

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Spotted a typo ;-) Also I'm not sure about `Astro CLI` vs `CLI d'Astro` but Astro should start with an uppercase.

Otherwise, LGTM!

**Comment by thomasbnt** [comment]
Thanks @ArmandPhilippot for reviews, they have been applied

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work! 🙌

**Pull request merged**